docs/library/network.rst

```diff
@@ -113,8 +113,46 @@ parameter should be `id`.
          connected to the AP.  The list contains tuples of the form
          (MAC, RSSI).
 
+.. method:: AbstractNIC.ipconfig('param')
+            AbstractNIC.ipconfig(param=value, ...)
+
+       Get or set interface-specific ip-configuration interface parameters.
+       Supported parameters are (availability subject to compile-time flags):
```

Comment:
I suggest "(availability of a particular parameter depends on the port and the specific network interface)".  Because users see the port/NIC and not so much which compile time flags are enabled.

Reply:
ooh yes, I changed the other occurrence. Now I should have both.